core/src/test/java/com/graphhopper/routing/util/EncodingManagerTest.java

```diff
@@ -66,6 +71,13 @@ public void testEncoderAcceptNoException()
     }
 
     @Test
+    public void testEncoderWithWrongVersionIsRejected()
+    {
+        thrown.expect(IllegalArgumentException.class);
```

Comment:
yeah, but not a high priority. maybe fix them when you stumble over them step by step :)

BTW: One more important issue for the testing stuff is a simple graph creation UI: https://github.com/graphhopper/graphhopper/issues/143